pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools", "wheel"]  # PEP 508 specifications.
+
+
+[tool.black]
+line-length = 120
+target_version = ['py37', 'py27', 'py36']
+exclude = '''
+
+(
+  /(
```

Comment:
Can you also exclude init template directories?

Reply:
good idea. sure can do.

Reply:
I have excluded the templates used for `sam init`. I will un-format them once, I do the update from develop and rerun black dance.